media/css/firefox/new/common/thanks-messaging.scss

```diff
@@ -18,12 +18,23 @@ $image-path: '/media/protocol/img';
 .show-windows,
 .show-windows-10-plus,
 .show-mac,
+.show-linux,
 .show-else,
 .show-linux-arm,
 .show-auto-download-notification {
     display: none !important;
 }
 
+.show-linux {
+    .c-linux-button-group {
+        margin-bottom: $spacing-md;
+
+        a:first-child {
+            margin-right: $spacing-md;
+        }
+    }
+}
+
```

Comment:
Issue: the buttons could use some slight alignment / spacing tweaks on /thanks? 

![image](https://github.com/mozilla/bedrock/assets/400117/af35dfbc-9d4e-49f7-b9fc-262d52292caa)